src/webview/webViewEventHandlers.js

```diff
@@ -15,8 +15,8 @@ import {
   messageLinkPress,
 } from '../actions';
 
-type MessageListEventReady = {
-  type: 'ready',
+type MessageListEventPing = {
```

Comment:
After some more thinking, while 'helo' is better than 'ping', 'ready' is even better. Let's keep it 'ready'.

This means no change to this file, and few undos/changes to the other files